src/compressed_tensors/offload/cache/cpu.py

```
@@ -17,13 +17,17 @@ class CPUCache(OffloadCache):
         super().__init__(onload_device, offload_device=offload_device)
 
-    def onload(self, offloaded: torch.Tensor | None) -> torch.Tensor | None:
+    def onload(self, key: str, offloaded: torch.Tensor | None) -> torch.Tensor | None:
         """
         Onload a tensor from cpu to device
 
+        :param key: parameter name (used to retrieve slicing information)
         :param key: cpu tensor to onload
         :return: device tensor
         """
+        slice = self.view_index.get(key, ...)
```

Ruff (0.15.21)

[error] Variable `slice` is shadowing a Python builtin

(A001)